hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/rate/Releasable.java

```diff
@@ -0,0 +1,5 @@
+package pl.allegro.tech.hermes.consumers.consumer.rate;
+
+public interface Releasable {
```

Comment:
i don't like the idea of this interface running around the code, can we make the name more meaningful? in our case it's not multi-purpose, we use it for inflight. I think we could make it `SubscriptionInflightLimiter` or sth like that. Now i see classes accepting `Releasable` and have no idea what/why it is there